### DISCUSSION TITLE
Bug of Use Claude

### DISCUSSION THREAD

**Discussion by LeckyHuang** [langgenius/dify#9305] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

我在dify上调用Claude的时候，提问第一句没问题，第二句开始就提示：
[anthropic] Bad Request Error, Error code: 400 - {'type': 'error', 'error': {'type': 'invalid_request_error', 'message': 'messages.3: all messages must have non-empty content except for the optional final assistant message'}}

**Comment by ytjhai** [comment]
I found this happens when there is only a "System" message block. Add a "User" message block with some content and then it will work. This happens because Claude is only a chat model not a completion model.
